woocommerce/woocommerce
# Fix undismissable notices when using localization for certain messages like "Coupon management has moved"

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Fix undismissable notice when using localization for certain messages like "Coupon management has moved"

Closes #36913.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Requisites: 
1. With WCA Test Helper, Set woocommerce_enable_coupons option to 'yes', 
2. Add the following string to `plugins/woocommerce/src/Admin/API/Options.php` $woocommerce_permissions variable: 
`wc_admin_show_legacy_coupon_menu` (needed to change the option in the next step)
3. set `wc_admin_show_legacy_coupon_menu` option  to '1'
4. Open the page /wp-admin/edit.php?post_type=shop_coupon&legacy_coupon_menu=1
3. You should see the message 'Coupon management has moved!'
4. Click the action.
5. The message should disappear.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Fix undismissable notice when using localization for certain messages like "Coupon management has moved"

#### Comment 

</details>
